packages/config/src/bridges/near.ts

```diff
@@ -64,23 +64,23 @@ export const near: Bridge = {
   },
   technology: {
     canonical: true,
-    category: 'Token Bridge',
+    category: 'Generic Bridge',
```

Comment:
We don't classify at the moment generic AMBs - in fact most bridges either are or use some AMB. The framework is - granted - a user-facing simplification and the current classification (TokenBridge, Liquidity Network, Hybrid) corresponds to the main use of the bridge